arangod/RocksDBEngine/RocksDBRecoveryManager.cpp

```diff
@@ -436,21 +431,17 @@ class WBReader final : public rocksdb::WriteBatch::Handler {
         return rocksdb::Status();
       }
 
-      if (coll->meta().countUnsafe()._committedSeq <= _currentSequence) {
-        auto& cc = coll->meta().countUnsafe();
-        cc._committedSeq = _currentSequence;
-        cc._added = 0;
-        cc._removed = 0;
-        coll->meta().loadInitialNumberDocuments();
-
-        for (std::shared_ptr<arangodb::Index> const& idx : coll->getIndexes()) {
-          RocksDBIndex* ridx = static_cast<RocksDBIndex*>(idx.get());
-          RocksDBCuckooIndexEstimator<uint64_t>* est = ridx->estimator();
-          TRI_ASSERT(ridx->type() != Index::TRI_IDX_TYPE_EDGE_INDEX || est);
-          if (est) {
-            est->clear();
-            est->setAppliedSeq(_currentSequence);
-          }
+      uint64_t currentCount = coll->meta().numberDocuments();
+      if (currentCount != 0) {
+        coll->meta().adjustNumberDocumentsInRecovery(_currentSequence, 0,
```

Comment:
In case the previous count is 0, we don't call `adjustNumberDocumentsInRecovery(...)` now. This is fine from the counter point of view, but just wondering if we should still call it to bump the sequence number forward for the collection.

Reply:
No need. If we crash before we sync the counter again, we'll re-recover this either way, and if we sync the counter before the crash, then we'll use an effective seq larger than this anyway and it'll be fine.

Reply:
Ok!